app/utils.py

```diff
@@ -74,6 +75,14 @@ def sanitize_email(email_address: str, not_lower=False) -> str:
     return email_address
 
 
+def sanitize_next_url(url: Optional[str]) -> Optional[str]:
+    if url is None or len(url) == 0:
```

Comment:
Nits: we can use `if not url` here which would take into account these 2 cases.

Reply:
Changed it :+1: